search.go

```diff
@@ -426,11 +433,12 @@ func NewSearchRequest(q query.Query) *SearchRequest {
 // By default results are ordered by score, descending.
 func NewSearchRequestOptions(q query.Query, size, from int, explain bool) *SearchRequest {
 	return &SearchRequest{
-		Query:   q,
-		Size:    size,
-		From:    from,
-		Explain: explain,
-		Sort:    search.SortOrder{&search.SortScore{Desc: true}},
+		ClientContextID: search.NewUUID(),
```

Comment:
Let's not do this by default here.

In bleve's purview it should remain something that's exclusively set by the client. So for couchbase, cbft will be responsible for generating a UUID if and only if the user does not already provide one and some config is enabled requiring us to generate for every request.